tarantool/connection.py

```diff
@@ -81,6 +81,7 @@
     InternalError,
     ProgrammingError,
```

Comment:
Moreover, it doesn't seems like connection_pool supports crud native API now. Choosing default modes may be a bit complicated, so feel free to ask if needed. Maybe we should provide a comment like "crud native API in pool makes sense only for pool consisting of crud routers" too.

Reply:
Done. AFAIU, I implemented test cases for `mesh connection` and `connection pool`, and now it works